### ISSUE TITLE
Warning on serve

### ISSUE THREAD

**Issue by vreuter** [databio/mkdocs-databio#13]
There are warnings for extra config keys like `pypi_name` and `site_logo`. Validation for those is done [within `Config`](https://github.com/mkdocs/mkdocs/blob/master/mkdocs/config/base.py#L68-L73). We may be able to alter the validation process in some way to accommodate our custom keys, or move them to the section for this plugin in a `mkdocs.yml` config file. We declare these in our own config schema, but it seems that at the time/level of the validation that `mkdocs` is doing, it's [only using a default schema](https://github.com/mkdocs/mkdocs/blob/master/mkdocs/config/base.py#L191-L193).

**Comment by nsheff** [comment]
yeah I think you can specify them in config and then it won't warn. I guess that's what your PR does right?

**Comment by vreuter** [comment]
No, there's still a warning, maybe because the keys are declared at top level in the config but are only known to the plugin's schema. We should probably localize the custom KV pairs to the plugin subsection of each config, since then there's no warning. I just don't know enough about HTML to use them if they're there rather than at top level. It'd be something like changing `config.site_logo` and `config.pypi_name` in the `base.html` and `main.html` files, I guess.

**Comment by vreuter** [comment]
In other words:
- top-level in config + curren which is odd since `mkdocs`'s own documentation t plugin state --> works on the frontent, but gives warning.
- plugin-section-local in config + `config.plugins.databio.pypi_name`, for instance --> doesn't work on frontend but warning is solved

**Comment by nsheff** [comment]
given that these variables are part of the theme and not part of the plugin...I guess we wouldn't move it into our plugin section, but maybe we can add it to a theme-specific section instead.
